lib/http.js

```diff
@@ -27,6 +28,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       instance: '/vc-issuer/instances/:instanceId',
       instanceIssuer: '/vc-issuer/instances/:instanceId/issuer',
       automatedIssue: '/vc-issuer/issue',
+      issueCredential: '/issuers/:issuerId/credentials',
```

Comment:
`issuerId` here is `instanceId`, please change the name to avoid confusion.

Reply:
instanceId https://github.com/digitalbazaar/bedrock-vc-issuer/pull/27/commits/b038742b4f4b5161fcba3d7c4da0168b24ef08ac